### DISCUSSION TITLE
Crowdin translations - need to repeat already translated

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#8670] [Firefly III]
Hi,

I am keeping Slovenian translation updated but lately I get messages about new strings to translate that I have already translated. Can you please check why?

Currently it is back to 100 %

![image](https://github.com/firefly-iii/firefly-iii/assets/13857043/4894f272-3259-45d9-a1e2-d6f9d991b05d)

Thank you.

**Comment by JC5** [comment]
Yes this sometimes happens when sentences or titles change, even a little. I see it in my own language as well. It's Crowdin's logic unfortunately, not mine.
